go-for-eat/localStorage.js

```diff
@@ -0,0 +1,21 @@
+export const loadState = () => {
+  try {
+    const serializedState = Expo.SecureStore.getItemAsync('state');
+    if (serializedState === null) return undefined;
+    return JSON.parse(serializedState);
+  } catch (err) {
+    console.log(err);
+    return undefined;
+  }
+};
+
+export const saveState = (state) => {
+  try {
+    const serializedState = JSON.stringify({
+      authentication: state.authentication,
```

Comment:
linting: if you use trailing commas use them always if not, don't use them ever. You know how you could avoid this? By setting up the linter correctly!!!!! ¬¬ Mike is not amused.